app/src/main/java/edu/uc/groupProject/topten/ui/main/MainFragment.kt

```diff
@@ -70,13 +62,13 @@ class MainFragment : Fragment() {
      */
     override fun onActivityCreated(savedInstanceState: Bundle?) {
 
-
         super.onActivityCreated(savedInstanceState)
 
+        activity.let {
```

Comment:
This is where instead of creating additional instances of ViewModels, the fragment makes use of already existing ones in an Activity.